extensions/amp-analytics/0.1/test/test-cookie-writer.js

```diff
@@ -292,6 +292,28 @@ describes.fakeWin('amp-analytics.cookie-writer value', {amp: true}, env => {
     });
   });
 
+  it('should write cookie with custom expiration', () => {
```

Comment:
Nice test! nit please also add test for invalid `cookieMaxAge` value. Thank you.